mat64/dense.go

```diff
@@ -360,6 +356,12 @@ func (m *Dense) Clone(a Matrix) {
 				copy(mat.Data[i*c:(i+1)*c], amat.Data[i*amat.Stride:i*amat.Stride+c])
 			}
 		}
+	case *Vector:
+		amat := aU.mat
+		mat.Data = make([]float64, aU.n)
```

Comment:
It actually could if someone constructed it that way. A matrix that stores its elements in a blas64.General but does not have a RawMatrix method would allow this. Probably I should allocate a new slice there since I am now saying that Clone **will not** cause shadowing.